carapace-server/src/main/java/org/carapaceproxy/server/config/RouteConfiguration.java

```diff
@@ -29,13 +34,13 @@
     private final String id;
     private final String action;
     private final boolean enabled;
-    private final RequestMatcher matcher;
+    private final String matchingCondition;
```

Comment:
Having a RequestMatcher means that we have pre-compiled the expression into a stateless java object, so we don't have to reparse the configuration string at every request